tools/release/sync-gutenberg-packages.js

```diff
@@ -0,0 +1,227 @@
+/* eslint-disable no-console */
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+const spawn = require( 'cross-spawn' );
+const { zip, uniq, identity, groupBy } = require( 'lodash' );
+
+/**
+ * Constants
+ */
+const WORDPRESS_PACKAGES_PREFIX = '@wordpress/';
+const { getArgFromCLI } = require( `../../node_modules/@wordpress/scripts/utils` );
+const distTag = getArgFromCLI( '--dist-tag' ) || 'latest';
+
+/**
+ * The main function of this task.
+ *
+ * It installs any missing WordPress packages, and updates the
+ * mismatched dependencies versions, e.g. it would detect that Gutenberg
+ * updated react from 16.0.4 to 17.0.2 and install the latter.
+ */
+function refreshDependencies() {
```

Comment:
Maybe let's change the name?
```suggestion
function main() {
```